### ISSUE TITLE
Add "reliability" to the TD

### ISSUE THREAD

**Issue by mlagally** [w3c/wot-thing-description#1002]
Similar to precision (see #1001) there are also sensors that are not very reliable,
i.e. deliver a false reading with a certain probability.

The TD should be able to indicate that characteristic.

**Comment by egekorkan** [comment]
Applies to both issues. I had the following diagram:
https://github.com/w3c/wot-architecture/issues/456#issuecomment-600869711

These terms, even in English, are not clear to everyone and everyone understands something slightly different. Thus, I would say that WoT Architecture has to define what these terms mean first.
On the other hand, some vocabulary should be able to describe this and we may not have to invent it ourself.

**Comment by relu91** [comment]
> On the other hand, some vocabulary should be able to describe this and we may not have to invent it ourself.

I agree. If we don't find any vocabulary/ontology that fits we could define them ourselves. However, I am wondering we might need to describe these concepts in other ontology, to no clutter the TD vocabulary. We could also move there the `unit` term, which belongs more to the "data" space rather than the affordances description.

**Comment by relu91** [comment]
I am thinking about this layered approach:
![image](https://user-images.githubusercontent.com/3170227/99062337-270d1d80-25a3-11eb-888e-c38189c4f5f2.png)

Just to explore the idea.

**Comment by danielpeintner** [comment]
Is there a relation with `stability` that has been used a LONG time ago (see https://github.com/w3c/wot-thing-description/issues/29)?
e.g., to indicate that a certain value does not change very often,

I agree both terms, `reliability `and `stability` might be part of an extended vocabulary.

**Comment by sebastiankb** [comment]
@mlagally 
> Similar to precision (see #1001) there are also sensors that are not very reliable,
i.e. deliver a false reading with a certain probability.

For me this are product specific information (of a sensor) which may be shared by the vendor in the TD. In the industry context specific metadata already available such as from eCl@ss.

**Comment by mlagally** [comment]
@danielpeintner 
"stability" as in #29 has a different definition.
In the context of this issue "reliability" indicates the probability for a correct sensor reading. It could be a percentage value, i.e. 100% means that each sensor readout is guaranteed to be accurate within the bounds defined by #1001

**Comment by mlagally** [comment]
@sebastiankb 
It would be useful to see a concrete eCl@ss example.
@relu91 
I think it is important to keep these concepts in the core vocabulary.
There were cases where serious accidents happened due to misinterpretation of values (e.g. different unit system).

**Comment by relu91** [comment]
> I think it is important to keep these concepts in the core vocabulary.
There were cases where serious accidents happened due to misinterpretation of values (e.g. different unit system).

Totally agree about the importance of these terms. Still, they belong to a different logical domain. So I proposed a layered approach to cleanly divide the "Interaction" from "data" domain. Could we just enforce the use of this new vocabulary layer in the TD core profile?

**Comment by mlagally** [comment]
@relu91 
Are you suggesting to move the unit (and potentially others) out of the current TD context?
Where would this new vocabulary layer be defined?

**Comment by relu91** [comment]
Yes, that's the idea. About where it should be defined I have two possibilities in mind:
- Within the TD taskforce. So it should be published in the TD specs, the only difference is that now we have two ontology files.
- As a totally different document published as Note.

**Comment by sebastiankb** [comment]
From today's TD call: 

It seems also a similar term in the SSN ontology. We should evaluate this.

**Comment by sebastiankb** [comment]
@mlagally 
>It would be useful to see a concrete eCl@ss example.

ok I will come back here and show an example

**Comment by sebastiankb** [comment]
ECLASS has different kind of "reliablity" description such as for [classes](https://www.eclasscontent.com/index.php?action=search&language=en&version=11.1&options=class&searchtxt=reliability) or [properties](https://www.eclasscontent.com/index.php?action=search&language=en&version=11.1&options=prop&searchtxt=reliability).

As you can see there are different contexts possible such as for [percent probability of a fault-free operation ](https://www.eclasscontent.com/index.php?action=cc2prdet&language=en&version=11.1&id=&pridatt=0173-1%2302-AAY978%23001) and [time interval associated with the reliability](https://www.eclasscontent.com/index.php?action=cc2prdet&language=en&version=11.1&id=&pridatt=0173-1%2302-AAY979%23001).

**Comment by sebastiankb** [comment]
From today's TD call:
- we need to clarify if this term should be used for the model and/or for value restrictions
- should be the number in percentage or have a time constraints
- @mlagally we need some more feedback in which context you need this kind of information. A use case would be good.

**Comment by vcharpenay** [comment]
The example I give in #1071 is also relevant here. SSN/SOSA has a module to specify in details the 'reliability' of a sensor (in terms e.g. of precision, resolution or accuracy, as in [Ege's diagram](https://github.com/w3c/wot-thing-description/issues/1002#issuecomment-726182422)).
